### ISSUE TITLE
Q: Is there an extension to create custom buttons?

### ISSUE THREAD

**Issue by Tset-Noitamotua** [squidfunk/mkdocs-material#160]
(This is just a question - not a bug issue)

I would like to place buttons in the docs to quickly trigger build jobs on a jenkins server. Do you know if there is an extension which could be used for that?

Or how can I make it myself?
My starting  point would be:
- Stackoverflow: [make-a-link-use-post-instead-of-get](http://stackoverflow.com/questions/3915917/make-a-link-use-post-instead-of-get)
- Jenkins API: Perform a build

Cheers
Tset

**Comment by squidfunk** [comment]
Well, you can just include a normal Markdown link and link to the Jenkins action. However, I don't know if that's possible with Jenkins.

**Comment by Tset-Noitamotua** [comment]
The link must trigger a `post` method thus one would have to use some JavaScript as in the stackoverflow link above.

**Comment by squidfunk** [comment]
You can you embed a plain HTML form in your Markdown with a button.

**Comment by Tset-Noitamotua** [comment]
Cool! Thank you!

I just added only this snippet into index.md

```html
<form name="jenkins_build" action="http://jenkins.mycompany.com/view/QS/job/QS_Project/build" method="post">
  <button>Run Test</button>
</form>
```

And the strage thing is - **it works!** I did not added any javascript to extra_js yet. How can that be?

Actually I am almost satisfied with that. Ony ugly thing is that a blank page opens in the same tab. I would like to stay in the tab and instead open the jenkins job's url in a new tab or window. An idea on how to achive that?

DISCLAIMER: I am kind of a noob in web dev.
